lib/foreman/cast.rb

```diff
@@ -0,0 +1,22 @@
+module Foreman
+  module Cast
+    def self.to_bool(value)
+      case value
+
+      when String
+        return true if value =~ (/^(true|t|yes|y|1)$/i)
+        return false if value.blank? || value =~ (/^(false|f|no|n|0)$/i)
```

Comment:
Could do, though having some consistency in acceptable values throughout the app might be good too.